scripts/manage.sh

```diff
@@ -7,10 +7,12 @@ cd "$LANGKIT_ROOT"
 make () {
     (
         cd "contrib/python"
+        rm build -rf
```

Comment:
Can you please add comments to explain why we need this? In particular, I’m curious why the mechanism that makes GPRbuild rebuild files when the runtime (and thus the compiler) has changed is not enough. Do you think this is a GPRbuild bug? Also, what is this “uncaught exception” you are referring to?

Reply:
Well personally I think the commit message is enough, but if it can put your mind at ease, I did say what I said in the commit message here again in a comment.

I'm not sure how much clearer I can be, so tell me if you still don't understand/need more precisions.

Reply:
It’s more about saving the trouble for future committers of looking in the commit history to understand where these comes from than putting my mind at ease. :-) The comments made crystal clear why removing the build directories is required, thank you very much.

Reply:
OK cool!

Reply:
Can you please re-approve then ? ;)

Reply:
Sure! Sorry, I forgot that you pushed new changes and thus needed that…